### PR TITLE
Update Genome assembly quality control

### PR DESCRIPTION
**Pull request by alexcorm**
Changes to be made after proofreading by Christophe Klopp and Delphine Naquin:

- [x] Slide 4: invert correctness and completeness
- [x] Slide 7: introduce L50 in addition to N50, talk about contig size distribution
- [ ] Slide 21: missing realignment of sequences on contigs to check that no pieces are missing. You can also look at the coverage along the contigs to see if there are areas that are more or less covered, indicating areas in multiple copies in the sequences but not in the assembly. 
- [x] Slide 24: on the illustration, it is not obvious why in the assembly there is a G while in the reads there are only A's or a deletion. The same goes for the G that follows, with 4 T's in the reads and a C...
- [x] Slide 29 to 30: take home messages are missing: several assemblies = need for criteria to choose the best = validation. Use several validations! Find the criteria that best suit to have the best genome for further analysis. Some criteria are expected at the time of publication.
- [x] I had to use the link in the slide to really understand what "yak hamming errors" were. I was probably missing the speech in the slide. Reverse red and blue.
- [x] Slide 18: a very heterozygous genome can have the same graph! But the assembly will be twice as big. 
- [x] Another point but it's not very clear in my head either. Shouldn't the terms "homozygous, heterozygous, haploid and diploid" be redefined for this notion of kmer? -> Delete graph KAT.
- [x] Slide18 to 21: I have trouble understanding the interest of showing the result of the 2 tools (Merqury and KAT).
- [x] Slide 15: busco is also limited because it only looks at "single copies" and we want a globally good genome (we can also use the transcriptome alignment of a close species to check the completeness of genes or a de novo RNA-Seq assembly of the same species).
- [x] Slide 13: I'm not sure if we are talking about core genes (rather used for the different genome assemblies of the same species, core-genome versus pan-genome)...to be checked.
- [x] Slide17: "TriCoanu" error.
- [x] Slide 16: What is "coherence"? The explanation of how to build the graph is missing.
- [x] Slides 14: what is the speech? - Make 2 figures instead of 2 and give the number of SG in Vertebral/Artho and Fungi
- [x] Slide 12: Lack of information on how to get the expected size, literature (genomesize.com), kmer (genomescope2 on reads), close species -> Reminder to the introduction (link).
- [x] Slide 11 to 12: Missing a slide for the different types of completeness: genome size, non-N content, gene content (linked to correctness), kmer content.
- [x] Slide 6: Add sequence definition.
- [x] Slide 3: Is my genome assembly ready for scaffolding and/or annotation -> 2 sentences instead of 1.

### CODE REVIEW

File: topics/assembly/tutorials/assembly-quality-control/slides.html [hunk 1]
```diff
@@ -215,57 +247,29 @@
...
-</br>
-
-.pull-left[
 .image-70[
 ![Merqury spectra-cn plot for highly heterozygous assembly and for which haplotig were purged or collapsed.](../../images/genome-qc/kmer-spectra-merqury-heterozygous-good.png)
-]]
-
-.pull-right[
-.image-70[
-![KAT spectra-cn plot for highly heterozygous assembly and for which haplotig were purged or collapsed.](../../images/genome-qc/kmer-spectra-kat-heterozygous-good.png)
```
Comment (hexylena) [inline topics/assembly/tutorials/assembly-quality-control/slides.html:267]: Is there a reason to remove all of the KAT images from the slide deck?

**Review by hexylena — COMMENTED** [review]
Overall this looks fine! I made some small changes, and one open question

File: topics/assembly/tutorials/assembly-quality-control/slides.html (hunk 1 above)
Comment (alexcorm) [reply topics/assembly/tutorials/assembly-quality-control/slides.html:267]: Yes, the wrapper for KAT is still not available and Merqury offers more functions to evaluate the quality of assemblies even if the full range of possibilities is not presented (trio, QV, switch,etc).

**Review by hexylena — APPROVED** [review]

**Comment by alexcorm** [comment]
@hexylena I did not complete all the tasks! Can you undo the merge so I can complete the changes?

**Comment by hexylena** [comment]
@alexcorm maybe it would be easier to just open a new branch, rather than reverting? And just add on to that? Or do you think it will change more and it would be better to revert, because the changes you already made looked good. (Apologies for the inconvenience!)

**Comment by alexcorm** [comment]
@hexylena don't worry, I will open a new branch and create the PR :)

**Pull request merged**